python/py-spdx/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx
+version             2.5.0
+revision            0
+
+maintainers         nomaintainer
+description         SPDX license list database
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             CreativeCommonsZerov1.0Universal
```

Comment:
Same as above.

Reply:
I was not able to find any licence in Macports list for this.
Should I edit it to CC0-1.0
https://creativecommons.org/publicdomain/zero/1.0/

Reply:
CC0 is not really a license but a public domain dedication. So say `public-domain`.

Reply:
Okey